tests/test_caching.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```

Comment:
In #57, I intentionally deleted these tests - the new tests don't need to explicitly run Wannier90. Also, it's not really aiida-wannier90's job to test the AiiDA caching feature - I think I just put this here initially because I was working on both in parallel.

Reply:
Same as above... my fault.